stytch/api/sessions.py

```diff
@@ -99,7 +103,10 @@ def authenticate_jwt(
             }
         except Exception as e:
             # JWT could not be verified locally. Check with the Stytch API.
-            return self.authenticate(session_jwt=session_jwt).json()
+            if session_custom_claims:
+                return self.authenticate(session_custom_claims=session_custom_claims, session_jwt=session_jwt).json()
+            else:
+                return self.authenticate(session_jwt=session_jwt).json()
```

Comment:
[pebble] I think `self.authenticate(session_custom_claims=None, ...)` should be able to handle this. Do we need to have this branch here for some other reason?

Reply:
[c87db63](https://github.com/stytchauth/stytch-python/pull/76/commits/c87db6321d6e9025a9734d288fade539ce851739)